db/migrate/20240924114730_add_taxes_deduction_rate_fields_to_fees_and_applied_taxes.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class AddTaxesDeductionRateFieldsToFeesAndAppliedTaxes < ActiveRecord::Migration[7.1]
+  def change
+    safety_assured do
+      change_table :fees do |t|
+        t.float :taxes_base_rate, default: 1.0, null: false
+      end
+
+      change_table :invoices_taxes do |t|
+        t.bigint :taxable_base_amount_cents, default: 0, null: false
+      end
+    end
```

Comment:
What about the following to avoid `safety_assured`?

```suggestion
    add_column :fees, :taxes_base_rate, :float, default: 0.0, null: false
    add_column :invoices_taxes, :taxable_base_amount_cents, :float, default: 0.0, null: false
```

Reply:
updated